test/integ/axpy/src/main.cpp

```diff
@@ -97,7 +98,7 @@ struct VectorAddKernelTester
         using Val = float;
 
         // Create the kernel function object.
-        VectorAddKernel kernel;
+        AxpyKernel kernel;
 
         // Get the host device.
         auto devHost(alpaka::dev::cpu::getDev());
```

Comment:
I can do this. While I am at it, should I directly remove the `cpu::getDev()` method because it only forwards to `getDevByIdx(0u)`?